ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc

```diff
@@ -117,10 +125,30 @@ typedef enum {
   TFLITE_DELEGATE_NNAPI,
   TFLITE_DELEGATE_XNNPACK,
   TFLITE_DELEGATE_EXTERNAL,
+  TFLITE_DELEGATE_QNN,
 
   TFLITE_DELEGATE_MAX
 } tflite_delegate_e;
 
+/**
+ * @brief Internal enum for QNN backend type.
+ */
+typedef enum QNNBackendType {
+  QNN_BACKEND_UNDEFINED = 0,
+  QNN_BACKEND_GPU,
+  QNN_BACKEND_HTP,
+  QNN_BACKEND_DSP
+} QNNBackendType;
+
+/**
+ * @brief Internal enum for QNN erformance mode.
```

Comment:
It looks like a typo.
```suggestion
 * @brief Internal enum for QNN performance mode.
```